source/_components/binary_sensor.mqtt.markdown

```diff
@@ -115,6 +115,35 @@ force_update:
   reqired: false
   type: boolean
   default: false
+device:
+  description: 'Information about the device this binary sensor is a part of to tie it into the [device registry](https://developers.home-assistant.io/docs/en/device_registry_index.html). Only works through [MQTT discovery](/docs/mqtt/discovery/) and when [`unique_id`](#unique_id) is set.'
+  required: false
+  type: map
+  keys:
+    identifiers:
+      description: 'A list of IDs that uniquely identify the device. For example a serial number.'
+      required: false
+      type: list, string
+    connections:
+      description: 'A list of connections of the device to the outside world as a list of tuples `[connection_type, connection_identifier]`. For example the MAC address of a network interface: `[["mac", "02:5b:26:a8:dc:12"]]`.'
```

Comment:
Should that be single, or double, brackets?

Reply:
double brackets. It's a list of 2-tuples.

Reply:
I clarified it a bit now by including `"connections": ` in the example config. I'm writing it in JSON format as that's the only way this config option can be used (only over MQTT discovery).